Maestro/subscriptions.json

```diff
@@ -850,6 +850,35 @@
         }
       }
     },
+    // Update dependencies in WCF release/2.1.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs/master/Latest.txt",
```

Comment:
This one was turned off in CoreFX: https://github.com/dotnet/corefx/blob/c61475013bc71f6d33f9da2694437adccb125588/dependencies.props#L96. (Don't know for sure if this needs to be matched in WCF though.)

Reply:
I believe it is used to get the latest version of the test ILC used for UWP runs. Is CoreFx not doing any UWP runs anymore?

Reply:
Yes, as Davis noted they were turned off, I think some stuff got broken and it will be a while til it's fixed.

Reply:
Ok, will remove it.

Reply:
Make sure to change it in WCF, since this file only controls when updates are triggered.